src/bullet_utils/wrapper.py

```diff
@@ -33,6 +34,22 @@ def __init__(
         self.joint_names = joint_names
         self.endeff_names = endeff_names
 
+        self.base_linvel_prev = None
+        self.base_angvel_prev = None
+        self.base_linacc = np.zeros((3, 1))
+        self.base_angacc = np.zeros((3, 1))
+
+        self.rng = default_rng()
+
+        self.base_imu_accel_bias = np.zeros((3, 1))
+        self.base_imu_gyro_bias = np.zeros((3, 1))
+        self.base_imu_accel_thermal = np.zeros((3, 1))
+        self.base_imu_gyro_thermal = np.zeros((3, 1))
```

Comment:
Is there a reason you are allocating arrays of size `(3, 1)`? Usually we try to go with one-dimensional arrays, so in this case only of size `np.zeros(3)`.

Reply:
I was having some numpy issues with matrix-vector products in another line, so I explicitly used column vectors everywhere to try to fix it. To be honest, though, I'm not great with numpy, so maybe this was unnecessary. The idea of using one-dimensional arrays instead of explicit column (3,1) or row (1,3) vectors is weird to me coming from Eigen. I also don't know what best practice is for numpy, or if this is more a preference thing?

I can definitely try to rewrite these commits using 1d arrays.